code/src/components/Details.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { NotFound } from './NotFound';
+
+export const Details = () => {
+  const [movieDetails, setMovieDetails] = useState([])
+
+  const { id } = useParams()
+
+  /* To round rating number to one decimal */
+  const roundedNumber = Math.round(movieDetails.vote_average * 10) / 10
+
+  const FetchDetails = () => {
+    fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=874b455d7037920aea9fd13db8645525&language=en-US`)
+      .then((response) => response.json())
+      .then((data) => setMovieDetails(data))
+      .catch((error) => {
+        console.log(error)
+        if (error) {
+          return (<NotFound />
+
+          )
+        }
+      })
+  }
+
+  useEffect(() => {
+    FetchDetails()
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  return (
+    <div
+      className="background"
+      style={{
+        backgroundImage: `url(http://image.tmdb.org/t/p/original${movieDetails.backdrop_path})`
+      }}>
+      <Link to="/" className="back-button">
+        <img src="./back-arrow-100.png" alt="Back button" />
+        <p>Back</p>
+      </Link>
+      <div className="movie-detail">
+        <img src={`http://image.tmdb.org/t/p/w342${movieDetails.poster_path}`} alt="movie" />
+        <div className="details">
+          <div className="title-vote">
+            <h1>{movieDetails.title}</h1>
+            <h3><span className="vote-icon" /> {roundedNumber}</h3>
```

Comment:
Adding to our comment on line 11: we really like that you made the math-round-up into a const. This makes the code very easy to read in the return.